src/app/static/src/app/store/modelCalibrate/mutations.ts

```diff
@@ -138,6 +139,10 @@ export const mutations: MutationTree<ModelCalibrateState> = {
 
     [ModelCalibrateMutation.CalibrateResultFetched](state: ModelCalibrateState, action: PayloadWithType<CalibrateResultResponse>) {
         state.result = action.payload
+    },
+
+    [ModelCalibrateMutation.ResetIds](state: ModelCalibrateState) {
```

Comment:
`ResetIds` seems a bit ambiguous (we have lots of ids!) - how about just `StopPolling` or `StopStatusPolling`?